cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,22 @@
+describe('template spec', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('Exibe a saudação "Hi, there" quando nenhum nome é fornecido', () => {
+    cy.get('input[data-testid="name"]').clear()
```

Comment:
Não há necessidade de limpar o campo, visto que seu estado inicial já é limpo.

Dê preferência pela simplicidade sempre.

```suggestion
    cy.get('input[data-testid="name"]')
```